lib/Announcement.dart

```diff
@@ -1,27 +1,421 @@
+                      child: DropdownButtonFormField(
+                        decoration: new InputDecoration(
+                          isDense: true,
+                          contentPadding: new EdgeInsets.symmetric(
+                              vertical: 10.0, horizontal: 10.0),
+                          border: OutlineInputBorder(
+                              borderRadius: BorderRadius.circular(32.0)),
+                          focusedErrorBorder: OutlineInputBorder(
+                            borderSide: BorderSide(color: Colors.black, width: 1.0),
+                          ),
+                          enabledBorder: OutlineInputBorder(
+                            borderSide: BorderSide(color: Colors.black, width: 1.0),
+                          ),
+                          labelText: "Whom",
+                        ),
+                        items: whom.map<DropdownMenuItem<String>>((e) {
+                          return (DropdownMenuItem(
+                            child: Text(e.toString()),
+                            value: e,
+                          ));
+                        }).toList(),
+                        value: whom[0],
+                        onChanged: (value) {
+                          setState(() {
+                            whomValue = value.toString();
+                            print(whomValue);
+                          });
+                        },
+                      )),
+                  Padding(
+                    padding: EdgeInsets.only(top: 10.0, bottom: 10.0,left: 60,right: 60),
+                    child: TextFormField(
+
+                      decoration: new InputDecoration(
+                        isDense: true,
+                        contentPadding:
+                        new EdgeInsets.symmetric(vertical: 10.0, horizontal: 0.0),
+                        border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(32.0)),
+                        focusedBorder: OutlineInputBorder(
+                          borderSide: BorderSide(color: Colors.black, width: 1.0),
+                        ),
+                        enabledBorder: OutlineInputBorder(
+                          borderSide: BorderSide(color: Colors.black, width: 1.0),
+                        ),
+
+                        prefixIcon: Padding(
+                          padding: EdgeInsets.all(0.0),
+                          child: IconButton(
+                            icon: const Icon(Icons.attach_file),
+                            onPressed: () {
+
+                            },
+                          ), // icon is 48px widget.
+                        ),
+                        hintText: 'Attachment',
+                        labelText: 'Attachment',
+                      ),
+                      validator: (val) {
+                        if (val == null || val.isEmpty) {
+                          return 'Please enter Description';
```

Comment:
no need to validate attachment, it can be empty

Reply:
ok